src/components/Calculator.js

```diff
@@ -1,48 +1,63 @@
 import React from 'react';
+// import operate from './operate';
```

Comment:
- [ ] For best practices, please delete this commented out line for cleaner and more optimized code 😊🙏🏻